tests/aggregation/tests.py

```diff
@@ -1727,6 +1728,28 @@ def test_aggregation_random_ordering(self):
             ordered=False,
         )
 
+    def test_order_by_aggregate_transform(self):
+        class Mod100(Mod, Transform):
+            def __init__(self, expr):
+                super().__init__(expr, 100)
+
+        sum_field = IntegerField()
+        sum_field.register_instance_lookup(Mod100, "mod100")
```

Comment:
Happy to use `django.tests.utils.register_lookup` instead. This felt like a nicer that didn't pollute global state and made use of the relatively recently introduced instance lookups feature.